doc/xml/daps_user_output.xml

```diff
@@ -277,4 +277,34 @@
    </para>
   </important>
  </sect1>
-</chapter>
+ <sect1 id="sec.daps.user.output.timestamp">
+ <title>Putting the actual date and time into the target document</title>
```

Comment:
* actual -> current (according to Doug) :wink: 
* Use title style here: "Putting the current Date and Time into the Target Document"